particula/aerosol_dynamics/parcel_test.py

```diff
@@ -18,21 +18,88 @@
     charge=1,
 )
 
+standard_environment = environment.Environment(
+    temperature=298 * u.K,
```

Comment:
Can you check if these units can go? They seemed not needed yesterday but I am generally not sure ...

Reply:
they work as long as you have called u = UnitRegistry() only once. I had problems yesterday getting < 300, Kelvin Kelvin>